front/app/containers/Admin/messaging/index.tsx

```diff
@@ -88,7 +87,7 @@ const MessagingDashboard = ({
     <>
       <NavigationTabs>
         {tabs.map(({ url, label }) => (
-          <Tab key={url} active={matchPathToUrl(url).test(pathname)}>
+          <Tab key={url} active={pathname.includes(url)}>
```

Comment:
Any reason to not use the new function here? Or flipped: can't we use this everywhere and do we need isTopBarNavActive?